QuickDeployment/QuickDeploymentEnvironment.md

```diff
@@ -1,52 +1,84 @@
-# How To Setup Quick Deployment Environment
+# 1. Chocolatey Quick Deployment Environment (QDE)
+
+## 1.1. Summary 
+
+This is an overview on the Chocolatey Quick Deployment Environment (QDE). It provides a single virtual machine appliance to be imported into your hypervisor-of-choice, which contains most of the various components of a Chocolatey organizational solution.
+
+![QDE Architechture](../images/quickdeploy/QDE-architecture.gif)
 
 <!-- TOC -->
 
-- [Summary](#summary)
-- [Setup](#setup)
-  - [VMDK](#vmdk)
-  - [Hyper-V](#hyper-v)
-- [More Articles](#more-articles)
-- [Related Articles](#related-articles)
+- [1. Chocolatey Quick Deployment Environment (QDE)](#1-chocolatey-quick-deployment-environment-qde)
+    - [1.1. Summary](#11-summary)
+    - [1.2. QDE Components](#12-qde-components)
+    - [1.3. Links](#13-links)
+    - [1.4. Related Articles](#14-related-articles)
+    - [1.4.1. FAQs](#141-faqs)
+        - [1.4.1.1. What does the QDE (Quick Deployment Environment) Include?](#1411-what-does-the-qde-quick-deployment-environment-include)
+        - [1.4.1.2. What do I need to do make this happen in my environment?](#1412-what-do-i-need-to-do-make-this-happen-in-my-environment)
+        - [1.4.1.3. How much time does this save?](#1413-how-much-time-does-this-save)
+        - [1.4.1.4. What if I have a larger environment? (> 1k nodes)](#1414-what-if-i-have-a-larger-environment--1k-nodes)
+        - [1.4.1.5. Can we brag about how fast we were able to get configured?](#1415-can-we-brag-about-how-fast-we-were-able-to-get-configured)
 
 <!-- /TOC -->
 
-## Summary
+## 1.2. QDE Components
 
-The quick deployment environment appliance provides an architecture that contains:
+The QDE appliance provides a unified architecture containing the following components:
 
-* Nexus (configured w/repositories)
-* Jenkins w/Internalizer Jobs
-* Chocolatey Central Management
-* Scripts for Internal Deployment
+* **Sonatype Nexus Repository OSS** (pre-configured with repositories) - This is your repository, where you'll be storing your Chocolatey packages (nupkg files).
+* **Jenkins** (w/Internalizer Jobs) - This is your automation engine, that lets you run tasks on-demand and on a schedule. It will host the PowerShell scripts to help you auto-internalize packages from the Chocolatey Community Repository
```

Comment:
Can we change '(w/Internalizer Jobs)' to '(pre-configured with internalizer jobs)' for consistency?